flake.nix

```diff
@@ -0,0 +1,34 @@
+{
+  description = "OSX-KVM allows to run macOS on Linux machines";
+
+  inputs.nixpkgs.url = "nixpkgs/22.11";
+  inputs.flake-utils.url = "github:numtide/flake-utils";
+  inputs.osx-kvm.url = "path:./";
```

Comment:
Is this really necessary, considering these files are already inside a flake?

Reply:
It shouldn't be. You can write `${self}/fetch-macOS-v2.py` instead of `${osx-kvm}/fetch-macOS-v2.py` etc. and remove this input.